README.md

```diff
@@ -71,28 +69,27 @@ Example usage:
     Run `git push heroku master` to create a new release using these buildpacks.
 
     $ cat Procfile
-    web:    bin/start-pgbouncer-stunnel bundle exec unicorn -p $PORT -c ./config/unicorn.rb -E $RACK_ENV
+    web:    bin/start-pgbouncer bundle exec unicorn -p $PORT -c ./config/unicorn.rb -E $RACK_ENV
     worker: bundle exec rake worker
 
     $ git push heroku master
     ...
     -----> Multipack app detected
     -----> Fetching custom git buildpack... done
-    -----> pgbouncer-stunnel app detected
-           Using pgbouncer version: 1.5.4-heroku
-           Using stunnel version: 5.08
-           Using stack version: cedar-14
+    -----> pgbouncer app detected
```

Comment:
i would be very +1 so send an email with instructions asking all dod-infra engineers to force-upgrade their shogun staging to use this branch; have we tested this on heroku-14,16 and 18?

Reply:
I thought I did but totally understand if folks missed it. My staging environment is already using it and i’m on 16. Part of the reasoning for this is to help this build pack work on 18 since there are a few issues with stunnel.